src/components/AllAlerts/index.js

```diff
@@ -198,104 +193,102 @@ class AllAlerts extends Component {
-                          ></div>
-                          <p className={styles.sevParagraph}>{this.state.fyiCount} Fyi</p>
-                        </Col>
-                      </Row>
-                    </Col>
-                  </div>
-                </Row>
-              </Container>
+                          <div className={styles.parentAlerts}>
+                            <div className={styles.yellow}></div>
+                            <div className={styles.sevParagraph}>
+                              {this.state.lowCount} Mid
+                            </div>
+                          </div>
+
+                          <div className={styles.parentAlerts}>
+                            <div className={styles.green}></div>
+                            <p className={styles.sevParagraph}>
+                              {this.state.fyiCount} Low
+                            </p>
+                          </div>
+                        </Row>
+                      </Col>
+                    </div>
+                  </Row>
+                </Container>
+              </div>
+            </div>
+            <span className={styles.alertUnderlineBold}></span>
+            <div className={styles.parentDiv}>
+              <div className={styles.filter}>
+                <span className={styles.boldHeader}>Filter: </span>{" "}
+                <select
+                  className={styles.selct}
+                  id="select"
+                  onChange={e => this.onChange(e)}
+                >
+                  <option default>All</option>
+
+                  {sliced.map((tag, i) => {
+                    return (
+                      <option value={tag} key={i}>
+                        {tag}
+                      </option>
+                    )
+                  })}
+                </select>
+              </div>
             </div>
-          </div>
-        </div>
-        {!filteredLength ? (
-          <h1>LOADING...</h1>
-        ) : (
-          <div>
-            {this.state.filtered.map((alert, i) => {
-              return (
-                <AlertCard
-                  key={i}
-                  title={alert.title}
-                  category={alert.category}
-                  content={alert.content}
-                  tags={alert.tags}
-                  severity={alert.severity}
-                />
-              )
-            })}
-          </div>
-        )}
-      </Accordion>
+            {!filteredLength ? (
+              <h1>LOADING...</h1>
+            ) : (
+              <div>
+                {this.state.filtered.map((alert, i) => {
+                  return (
+                    <AlertCard
+                      key={i}
+                      title={alert.title}
+                      category={alert.category}
+                      content={alert.content}
+                      tags={alert.tags}
+                      severity={alert.severity}
```

Comment:
@kem247 the mitigation field in the alert object is not being passed in here, so all of the alerts are getting the default currently. We should make sure it's being passed in here as a prop.

Reply:
I've already done that, I just haven't done another PR